sw/device/silicon_creator/lib/error.h

```diff
@@ -21,10 +21,11 @@ extern "C" {
  */
 enum module_ {
   kModuleUnknown = 0,
-  kModuleUart = 0x4155,       // ASCII "UA".
-  kModuleHmac = 0x4d48,       // ASCII "HM".
-  kModuleSigverify = 0x5653,  // ASCII "SV".
-  kModuleOtp = 0x504f,        // ASCII "OP".
+  kModuleUart = 0x4155,         // ASCII "UA".
```

Comment:
Do you think a macro like the following makes sense?


```c
#define OT_ASCII16(s) \
    (((uint16_t) ("" s "")[0]) | \
    (((uint16_t) ("" s "")[1]) << 8))
```
(The `"" s ""` idiom guarantees a string constant.)

We can just throw it in `macros.h` or whatever.

Reply:
I think I prefer to have these constants in `error.h` but I don't have a strong opinion on this. @cfrantz any thoughts?

Reply:
I prefer the hex constants because that is how the error codes will be printed.  If, OTOH we were printing out the two-letter ascii characters, I'd prefer the string-and-macro scheme.

Reply:
We can switch to the constants if we decide we like them better. *shrug*